crates/telio-batcher/src/batcher.rs

```diff
@@ -3,8 +3,18 @@ use std::fmt::Debug;
 use std::hash::Hash;
 use std::{collections::HashMap, sync::Arc};
 
-use telio_utils::telio_log_debug;
-use tokio::time::sleep_until;
+use telio_utils::{telio_log_debug, telio_log_warn};
+use tokio::time::{sleep_until, Duration};
+
+// Batcher is an optimizer of aligning actions to be done in batches
+// with the goal to conserve resources, thus Batcher's main goals are:
+// 1) Don't break existing functionality and limit damage if possible.
+// 2) Align actions.
+
+// It's crucial not to allow too short intervals since they can drain resources.
+// Historically, libtelio had highest frequency action of 5seconds for direct
+// connection keepalive, thus we cap any action to have at least that.
+const MIN_INTERVAL: Duration = Duration::from_secs(5);
```

Comment:
maybe it could be an associated const?

Reply:
Could you clarify what do you mean by that in this context? Should I add the context to the struct somehow?